mule/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: mule
+    version: 1.2
+
+source:
+    fn: mule.tar.gz
+    url: https://github.com/SciTools/mule/releases/download/vn1.2/mule.tar.gz
+
+build:
+    number: 0
+    skip: True # [py3k or win]
```

Comment:
@stevewardle I think this may have to be an and instead of an or:

skip: True # [py3k and win]